aether-kernel/aether/kernel/api/filters.py

```diff
@@ -165,7 +165,7 @@ def mapping_filter(self, queryset, name, value):
             return queryset.filter(mappings__name=value)
 
     class Meta:
-        fields = '__all__'
+        exclude = ('topic',)
```

Comment:
Do we need to exclude the topic filed in filters? If we need it to be used in the producer it looks like that we need to filter the entries by this field...

Reply:
:woman_shrugging:

Reply:
The producer is going to use a single query that doesn't take topic into account. Don't need the filter for that specific purpose.